vendor/fasttext/curlstream.cc

```diff
@@ -0,0 +1,132 @@
+#include <memory>
+#include <chrono>
+#include <thread>
+#include <streambuf>
+#include "./curlstream.h"
```

Comment:
Is this `./` prefix necessary?

Reply:
Not strictly, but it's in case a file of the same name existed in the include paths.